chart/values.yaml

```diff
@@ -140,9 +140,38 @@ updateStrategy:
 volumeMounts:
   cache:
     mountPath: /var/cache/nginx
+  custom-networks-config:
+    mountPath: /app/networks-config.json
+    subPath: networks-config.json
 
 # Volume mounts to add to the container. The key is the volume name and the value is the volume definition. Evaluated as a template.
 volumes:
   cache:
     emptyDir: {}
 
```

Comment:
Extra new line here between other volumes. Not sure if it causes problems in templating.